pkg/client/src/app/pages/identities/components/identity-form.tsx

```diff
@@ -74,17 +126,17 @@ export const IdentityForm: React.FC<IdentityFormProps> = ({
     encrypted: identity?.encrypted || "",
     id: identity?.id || 0,
     key: identity?.key || "",
-    kind: { value: "", toString: () => "" },
-    userCredentials: { value: "", toString: () => "" },
+    kind: kindInitialValue,
```

Comment:
I might be missing something but why kind cannot be just initialized with `""` ?
Having the field mandatory should suffice because the user can only select its value through a drop-down box which we control the value of. In which case we wouldn't need `toString()`